web/src/components/game/goals/GoalUpload.tsx

```diff
@@ -156,25 +71,54 @@ export default function GoalUpload({ isOpen, close, slug }: GoalUploadProps) {
                                         ))}
                                     </Tab.List>
                                     <Tab.Panels className="mt-2 h-full">
-                                        <Tab.Panel className="h-full rounded-xl p-3">
-                                            <div className="mb-2.5 flex items-center gap-x-3 rounded-md bg-yellow-300 px-2 py-1 text-sm text-black">
+                                        {uploadModes.map((tab) => (
+                                        <Tab.Panel key={tab} className="h-full rounded-xl p-3">
+                                            {tab === 'List' && (
                                                 <div>
-                                                    <FontAwesomeIcon
-                                                        icon={faExclamation}
-                                                        className="mt-1 rounded-full border border-black px-3 py-1 text-lg"
+                                                    <div className="mb-2.5 flex items-center gap-x-3 rounded-md bg-yellow-300 px-2 py-1 text-sm text-black">
+                                                        <div>
+                                                            <FontAwesomeIcon
+                                                                icon={faExclamation}
+                                                                className="mt-1 rounded-full border border-black px-3 py-1 text-lg"
+                                                            />
+                                                        </div>
+                                                        <div>
+                                                            Please ensure that the list
```

Comment:
If you want this text broken up when displayed you ned to break it into other blocks (probably with \<p\> blocks)